1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/exploit/ChatBypass

```diff
@@ -0,0 +1,46 @@
+package net.ccbluex.liquidbounce.features.module.modules.exploit;
+ 
+import net.ccbluex.liquidbounce.event.EventTarget;
+import net.ccbluex.liquidbounce.event.PacketEvent;
+import net.ccbluex.liquidbounce.features.module.Module;
+import net.ccbluex.liquidbounce.features.module.ModuleCategory;
+import net.ccbluex.liquidbounce.features.module.ModuleInfo;
+import net.minecraft.network.play.client.C01PacketChatMessage;
+ 
+@ModuleInfo(name = "ChatBypass", description = "Allows you to bypass chat sensors.", category = ModuleCategory.EXPLOIT)
+public class ChatBypass extends Module{
+    @EventTarget
+     public void onPacket(final PacketEvent event) {
+        if(event.getPacket() instanceof C01PacketChatMessage) {
+            final C01PacketChatMessage chatMessage = (C01PacketChatMessage) event.getPacket();
+            final String message = chatMessage.getMessage();
+            final StringBuilder stringBuilder = new StringBuilder();
+            final String replaced = "";
+           
+            for(char c : message.toCharArray()) {
+                if(c >= 33 && c <= 128) {
+                    replaced.replace(c, replaceChar(c));
+                }
+                else {
+                    replaced += c;
+                }
+            }
+            chatMessage.message = replaced;
+        }
+    }
+      public char replaceChar(char c) {
+          String superscriptCharMap[] = {"0","⁰","1","¹","2","²","3","³","4","⁴","5","⁵","6","⁶","7","⁷","8","⁸","9","⁹","a","ᵃ","b","ᵇ","c","ᶜ","d","ᵈ","e","ᵉ","f","ᶠ","g","ᵍ","h","ʰ","i","ⁱ","j","ʲ","k","ᵏ","l","ˡ","m","ᵐ","n","ⁿ","o","ᵒ","p","ᵖ","q","q","r","ʳ","s","ˢ","t","ᵗ","u","ᵘ","v","ᵛ","w","ʷ","x","ˣ","y","ʸ","z","ᶻ","A","ᴬ","B","ᴮ","C","ᶜ","D","ᴰ","E","ᴱ","F","ᶠ","G","ᴳ","H","ᴴ","I","ᴵ","J","ᴶ","K","ᴷ","L","ᴸ","M","ᴹ","N","ᴺ","O","ᴼ","P","ᴾ","Q","Q","R","ᴿ","S","ˢ","T","ᵀ","U","ᵁ","V","ⱽ","W","ᵂ","X","ˣ","Y","ʸ","Z","ᶻ","+","⁺","-","⁻","=","⁼","(","⁽",")","⁾"};
+          //Update superscriptCharMap to hashmap
+          char update;
+          String compare = String.valueOf(c);
+          for(int i = 0; i < superscriptCharMap.length - 1; i++) {
+             
+              if (compare == superscriptCharMap[i]) {
```

Comment:
You are comparing Strings using `==` rather than `equals`, please read https://www.geeksforgeeks.org/difference-equals-method-java/

Reply:
Also as @superblaubeere27 said better to just use chars, you can compare chars using `==` safely